### ISSUE TITLE
Command line prints out incorrect output file name

### ISSUE THREAD

**Issue by lyndondrake** [htdebeer/pandocomatic#70]
If I run pandocomatic, it gives the input file name as the output file name it has written to. Despite this, it actually writes to a different output file name, correctly. It's just a confusing message:

```
~/D/W/W/T/pandocomatic $ pandocomatic pandocomatic-test.md
Pandocomatic needed 1.1 seconds to convert '/Users/lyndon/Documents/Writing/Workflow/Test/pandocomatic/pandocomatic-test.md' to 'pandocomatic-test.md'.
```

Despite the output, it has actually written to pandocomatic-test.tex (which is what I waned and expected.

**Comment by htdebeer** [comment]
This does look like a bug. Can you add a minimal version of `pandocomatic-test.md` so I can reproduce the issue on my end?

**Comment by lyndondrake** [comment]
```
---
  author: Lyndon Drake
  title: "Test document"
  institute: University of Oxford
  date: 28 November 2018
  abstract: A little test abstract

  pandocomatic_:
    use-template: academic-article
---


This is a test document.
```

**Comment by lyndondrake** [comment]
The YAML file has changed a bit since I wrote the test document, but once I've got it working again I can post that too.

**Comment by htdebeer** [comment]
Please do, for I need that as well to reproduce the issue.

**Comment by lyndondrake** [comment]
```templates:
  general:
    pandoc:
      standalone: true
      pdf-engine: lualatex
      bibliography: /Users/lyndon/Documents/Writing/Workflow/bibliography.json
    metadata:
      papersize: a4
      fontsize: 10pt
      lang: en-GB

  latex-defaults:
    pandoc:
      biblatex: true
    metadata:
      bibliography: /Users/lyndon/Documents/Writing/Workflow/bibliography.bib
      biblatexoptions:
        - backend=biber
        - indexing=cite
      mainfont: Skolar PE
      mainfontoptions: Ligatures=TeX, Numbers=OldStyle
      mainfontoptions: Ligatures=TeX, Numbers=OldStyle


  academic:
    extends: ['general']
    pandoc:
      numbersections: true
      csl: /Users/lyndon/Documents/Writing/Workflow/csl-styles/society-of-biblical-literature-fullnote-bibliography.csl
      biblio-style: sbl


  academic-article:
    extends: ['academic','latex-defaults']
    pandoc:
      to: pdf
      table-of-contents: false
    metadata:
      documentclass: scrartcl

  academic-book:
    extends: ['academic']
    pandoc:
      table-of-contents: true
    metadata:
      documentclass: scrbook
```

**Comment by htdebeer** [comment]
Fixed the bug with release 2.5.2.
